packages/dappmanager/src/modules/stakerConfig/stakerParamsByNetwork.ts

```diff
@@ -65,7 +65,7 @@ export function stakerParamsByNetwork<T extends Network>(
             minVersion: ""
           }
         ],
```

Comment:
Can we include logic for handling erigon? At least on another parallel branch to test it on.

Reply:
Only tested, polished and published packages can be added in the stakers UI. However we can already take erigon into account when building the consensus clients for the stakers UI

Reply:
of course, sorry i really meant we should keep it in mind currently by not just hardcoding nethermind xdai for everything and should just have a parallel branch with it included for testing erigon with the gnosis stakersUI